src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Action to perform when double clicking on a password in the list view'      : 'Action exécutée lorsque vous double-cliquez sur un mot de passe dans la liste',
+            'Shows options to copy the password and user name in the menu'               : 'Affiche les options pour copier le mot de passe et le nom d’utilisateur dans le menu.',
+            'The initial section to be shown when the app is opened'                     : 'La section qui sera affichée à l’ouverture de l’application',
+            'Always show the username related to the password in the list view'          : 'Toujours afficher le nom d’utilisateur lié au mot de passe dans la liste',
+            'Show the tags for each password in the list view. Increases loading times'  : 'Affiche les balises pour chaque mot de passe dans la vue liste. Augmente le temps de chargement',
+            'Show hidden custom fields in the edit form and detail section of a password': 'Affiche les champs personnalisés cachés dans le formulaire de modification et dans les détails du mot de passe',
+            'Start search when a key is pressed anywhere on the page'                    : 'Commence la recherche dès qu’une touche est enfoncée',
+            'Search everywhere when the enter key is pressed in the search box'          : 'Effectuez une recherche n’importe où lorsque vous appuyez sur la touche Entrée dans la zone de recherche',
+            'Always show the section for global search in the navigation'                : 'Toujours afficher la section de recherche globale dans la navigation',
+            'Sends you e-mails about compromised passwords and other security issues'    : 'Vous envoie un courriel si un mot de passe est compromis ou que des problèmes de sécurité se produisent',
+            'Sends you e-mails when other people share passwords with you'               : 'Vous envoie un courriel quand des mots de passe vous sont partagés',
+            'Notifies you about compromised passwords and other security issues'         : 'Envoie une notification si un mot de passe est compromis ou que des problèmes de sécurité se produisent',
+            'Notifies you when other people share passwords with you'                    : 'Envoie une notification quand des mots de passe vous sont partagés',
+            'Notifies you when a background operation fails'                             : 'Envoie une notification quand un tache en arrière plan échoue',
+            'Reset all settings on this page to their defaults'                          : 'Réinitialise tous les paramètres de cette page à leur valeur de défaut',
+            'This will reset all settings to their defaults. Do you want to continue?'   : 'Cela réinitialisera tous les paramètres à leurs valeurs par défaut. Voulez-vous continuer ?',
+            'Start over and delete all configuration, passwords, folders and tags'       : 'Restaurer les paramètres d’usine et supprimer tous les mots de passe, dossiers et tags',
```

Comment:
Should be `Restaurer les paramètres d’usine et supprimer tous les mots de passe, dossiers et étiquettes`